_publications/jors/paper.tex

```diff
@@ -0,0 +1,362 @@
+present a quantitative comparison of past attempts in a compelling
+way. An exemplar of such a portal is
+$\mu$MAG~\cite{mumag}. The reusable PFHub
+framework leverages existing online services to build a static portal
+website that is considerably easier to deploy and maintain without
+sacrificing content or scope. The first deployment of the PFHub
+framework supports phase-field practitioners and code developers
+participating in an effort to improve quality assurance for
+phase-field codes.
+
+%% ~\footnote{Certain commercial equipment,
+%%   instruments, or materials (or suppliers, or software, ...) are
+%%   identified in this paper to foster understanding. Such
+%%   identification does not imply recommendation or endorsement by the
+%%   National Institute of Standards and Technology, nor does it imply
+%%   that the materials or equipment identified are necessarily the best
+%%   available for the purpose.\label{disclaimer}}
+
+%% ~\textsuperscript{\ref{disclaimer}}
+
+\section*{Keywords}
+
+phase-field; materials-science; jekyll-website; reproducible-science
+
+\section*{Introduction}
+
+The phase-field method (PFM) describes material interfaces at the
+mesoscopic scale between atomic scale models and macroscale models.
+The PFM is well established and there are an assortment of code
+frameworks (\emph{e.g.}, MOOSE~\cite{moose},
+PRISMS-PF~\cite{prisms-pf}, FiPy~\cite{fipy},
+MMSP~\cite{mmsp}) available for
+solving the wide variety of mesoscale phenomena (\emph{e.g.},
+dendritic growth, spinodal decomposition, grain growth). However,
+phase-field research groups often develop codes in isolation and do
+not support or distribute the code bases to the wider community. PFHub
+is a community effort spearheaded by the Center for Hierarchical
+Materials Design at Northwestern University and the National Institute
+of Standards and Technology to support the development of phase-field
+codes. In particular, the PFHub effort focuses on improving
+cross-collaboration between phase-field code developers and
+practitioners by providing a standardized set of benchmark
+problems~\cite{bm1, bm2} along with a web framework for uploading and
+comparing benchmark results from different codes.
+
+\section*{Implementation and architecture}
+
+The PFHub website provides a facility for uploading, displaying and
+comparing results from the benchmark problems. The website uses the
+Jekyll static website generator~\cite{jekyll} along with automated
+frontend processing to eliminate the need for content management
+systems~\cite{cmsfree}, which are generally costly to maintain
+especially for small scientific communities with limited funding and
+staffing.
+
+The workflow for uploading benchmark results relies on third party
+tools using the following steps, illustrated in
+Figure~\ref{fig:pfhub_website}.
+
+\begin{itemize}
+  \item The users are first required to upload simulation outputs to
+    an archival resource (\emph{e.g.}, Figshare~\footnote{Certain
```

Comment:
I am thinking all of these different softwares (broadly speaking) need references: Jekyll, Figshare, etc...

Reply:
Both have citations now.